### DISCUSSION TITLE
Help with .zip (Corrupted file)

### DISCUSSION THREAD

**Discussion by Felis14** [FunkinCrew/Funkin#1509] [Q&A]
I've tried like 7 times to download the file, between 150Mb to 170 Mb it stops, and if try to unzip it, WinRar says is a corrupted file, ¿What should I do?

**Comment by SuperSpinDashGamer39** [comment]
idk just delete some stuff that happens to me too and I just delete some stuff and it helps

**Comment by TentaRJ** [comment]
> I've tried like 7 times to download the file, between 150Mb to 170 Mb it stops, and if try to unzip it, WinRar says is a corrupted file, ¿What should I do?

Did you download it from the Itch.io page? Make sure it is the legitimate copy.

**Comment by Windows1Door2Daniel** [comment]
Remeber

you can use your OS File explorer to extract .zip folders
